panda_moveit_config/config/fake_controllers.yaml

```diff
@@ -21,3 +12,6 @@ controller_list:
     joints:
       - panda_finger_joint1
       - panda_finger_joint2
+initial:
+  - group: panda_arm_hand
```

Comment:
`ready` is for `panda_arm`

Reply:
Both `panda_arm` and `panda_arm_hand` have a `ready` `group_state` in the SRDF.